src/Shared/LogMessagePacketBase.cs

```diff
@@ -403,6 +409,7 @@ internal void ReadFromStream(ITranslator translator)
                 else
                 {
                     _buildEvent = ReadEventFromStream(_eventType, translator);
+                    ErrorUtilities.VerifyThrow(_buildEvent is not null, "Not Supported LoggingEventType {0}", _eventType.ToString());
```

Comment:
Kind of scary to just throw here, perhaps log an error instead? Or do we think that this will receive enough testing before the release?

Reply:
sure